package.json

```diff
@@ -90,12 +90,11 @@
   "size-limit": [
     {
       "path": "lib/lib.css",
-      "limit": "16 kB",
-      "webpack": false
```

Comment:
Out of interest (and because I couldn't wrap my head around the [size-limit readme](https://github.com/ai/size-limit#size-limit-) explanation) - why did we have `webpack: false` for the css to begin with and why can it be removed now?

Reply:
The previous preset default would seemingly load all your JS into a blank webpack project as a size test. That didn't work for CSS so they told you to disable it. We only ever cared about the file-size check so I'm including a simpler preset now which just checks gzipped size.